sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,14 +1,11 @@
 # Release History
 
-## 4.5.1 (Unreleased)
-
-### Features Added
+## 4.6.0 (2022-09-12)
```

Comment:
Ah, here and elsewhere, please update to 9/19 or 9/20 (we plan to slip to 9/20 for the announcement).